portability-core/src/main/java/org/dataportabilityproject/cloud/local/LocalCloudFactory.java

```diff
@@ -46,21 +61,35 @@ public JobDataCache load(String service) throws Exception {
         }
       });
 
-  private static final Supplier<PersistentKeyValueStore> KEY_VALUE_SUPPLIER =
-      Suppliers.memoize(InMemoryPersistentKeyValueStore::new);
+  // Google DataStore emulator is used for local development
+  private static final Datastore datastore = DatastoreOptions.newBuilder()
+      .setHost("http://localhost:8081")
```

Comment:
should all ports be moved to yaml so we avoid conflicts?

Reply:
Added TODO